artbotlib/pr_in_build.py

```diff
@@ -290,7 +290,7 @@ async def check_nightly_or_releases(self, releases: Iterable) -> str:
 
         earliest = None
         for release in releases:
-            if release['phase'] == 'Failed':
+            if release['phase'] == 'Failed' or release['phase'] == 'Pending':
```

Comment:
ok but this would be a little more concise:
```suggestion
            if release['phase'] in ('Failed', 'Pending'):
```